pkg/accounts/cli/accounts.go

```diff
@@ -75,11 +75,6 @@ func NewAccounts(c Config, l *zap.SugaredLogger) (*Accounts, error) {
 	return cli, nil
 }
 
-// WithAccountManager sets the account manager for the Accounts.
-func (c *Accounts) WithAccountManager(m accounts.Interface) {
-	c.accountManager = m
-}
-
```

Comment:
Are you sure this is unused? 🤔

Reply:
nevermind, it is unused. There's another `WithAccountManager` in the session package that I got confused with - https://github.com/percona/everest/blob/main/pkg/session/manager.go#L64